pkg/pool-weighted/contracts/CustomFeeAuthorizer.sol

```diff
@@ -0,0 +1,40 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.7.0;
+
+import "@balancer-labs/v2-solidity-utils/contracts/openzeppelin/Ownable.sol";
+
+contract CustomFeeAuthorizer is Ownable {
+    event authorizorAdded(address indexed _authorizor);
+    event authorizorRemoved(address indexed _authorizor);
+
+    mapping(address => bool) private authorized;
+    bool public isCustomFeeEnabled = false;
```

Comment:
I'm not sure we need this. I think it's safe to say owners will always want this to be enabled.